lib/template.html.eex

```diff
@@ -1,9 +1,17 @@
-<script src="https://www.google.com/recaptcha/api.js?hl=<%= @options[:hl] %>" ></script>
+<script src="https://www.google.com/recaptcha/api.js?onload=onLoad&hl=<%= @options[:hl] %>" async defer></script>
 
 <%= if @options[:size] == "invisible" do %>
 <script type="text/javascript">
   var form = null;
 
+  function onLoad() {
```

Comment:
this is heavily relying on the assumption that there is a submitbutton, and that this script is allowed to enable it. If other validations are also controlling disabled state (like form validations) this can confuse that behavior

Reply:
Good point, I will look into changing it.